### PR TITLE
MD Updates, Fix encounter typo

### PR DESCRIPTION
**Pull request by ssayer**
* Update definitions based on MD feedback
* Capitalize Encounter category in Inpatient Encounter
* Fix references to ICD9/ICD10 for all data elements

### CODE REVIEW

File: cimpl/ecqm_dataelement.txt [hunk 1]
```diff
@@ -273,7 +273,7 @@ ResultValue from https://vsac.nlm.nih.gov/valueset/2.16.840.1.113883.3.526.3.124
...
 Based on: PrincipalDiagnosis
-Description: "(Clinical Focus: This grouping value set contains diagnoses used to identify patients with an acute or evolving myocardial infarction.),(Data Element Scope: The intent of this data element is to identify patients with an acute or evolving myocardial infarction. Using the Quality Data Model, this particular element would map to the Result attribute.),(Inclusion Criteria: Include codes that identify patients with an acute myocardial infarction. This is a grouping of ICD10CM and SNOMEDCT codes.),(Exclusion Criteria: None.)"
+Description: "(Clinical Focus: This grouping value set contains diagnoses used to identify patients with an acute or evolving myocardial infarction.),(Data Element Scope: The intent of this data element is to identify hospital environments where a patient may receive inpatient or outpatient care. Using the Quality Data Model, this value set is used with the \"Admission Source\" attribute of the \"Encounter\" category.),(Inclusion Criteria: Include codes that represent the concept of hospital environment. Codes included are SNOMED CT 22232009 and all its children. ),(Exclusion Criteria: None.)"
```
Comment (okeefm) [inline cimpl/ecqm_dataelement.txt]: I think there may have been a mis-copy-paste. I edited in what I believe to be the correct wording below.

```suggestion
Description: "(Clinical Focus: This grouping value set contains diagnoses used to identify patients with an acute or evolving myocardial infarction (ST-segment elevation MI).),(Data Element Scope: The intent of this data element is to identify patients with an acute or evolving myocardial infarction (ST-segment elevation MI). Using the Quality Data Model, this value set is used with the “Principal Diagnosis” attribute of the “Encounter” category.),(Inclusion Criteria: Include codes that identify diagnosis of acute or evolving myocardial infarction (ST-segment elevation MI). This is a grouping of ICD-10-CM and SNOMED CT codes.),(Exclusion Criteria: None.)"
```

File: cimpl/ecqm_dataelement.txt [hunk 2]
```diff
@@ -4294,7 +4294,7 @@ Subject value is type HospitalObservationCareInitial
...
 Based on: AdmissionSource
-Description: "(Clinical Focus: This set of values identifies hospital environments where a patient may receive inpatient or outpatient care.),(Data Element Scope: The intent of this data element is to identify hospital environments where a patient may receive inpatient or outpatient care.  Using the Quality Data Model, this particular element will map to the Transfer category.),(Inclusion Criteria: Use the concept of hospital environment (22232009) and all its children.  Codes used are to be SNOMED CT codes only.),(Exclusion Criteria: None.)"
+Description: "(Clinical Focus: This set of values identifies hospital environments where a patient may receive inpatient or outpatient care.),(Data Element Scope: The intent of this data element is to identify hospital environments where a patient may receive inpatient or outpatient care. Using the Quality Data Model, this value set is used with the \"Admission Source\" attribute of the \"Encounter\" category.),(Inclusion Criteria:  Include codes that represent the concept of hospital environment. Codes included are SNOMED CT 22232009 and all its children. ),(Exclusion Criteria: None.)"
```
Comment (okeefm) [inline cimpl/ecqm_dataelement.txt]: The SHR toolchain doesn't like the escaped double quotes, so we'll probably have to change them to single quotes.
```suggestion
Description: "(Clinical Focus: This set of values identifies hospital environments where a patient may receive inpatient or outpatient care.),(Data Element Scope: The intent of this data element is to identify hospital environments where a patient may receive inpatient or outpatient care. Using the Quality Data Model, this value set is used with the 'Admission Source' attribute of the 'Encounter' category.),(Inclusion Criteria:  Include codes that represent the concept of hospital environment. Codes included are SNOMED CT 22232009 and all its children. ),(Exclusion Criteria: None.)"
```

File: cimpl/ecqm_dataelement.txt [hunk 3]
```diff
@@ -5225,7 +5225,7 @@ ResultValue from https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1046.6/
...
+Description: "(Clinical Focus: This set of values contains diagnoses that are commonly used in the inpatient setting for newborn live born, and born in hospital. This value set is a grouping value set composed of  ICD-10-CM and SNOMED CT value sets.),(Data Element Scope: The intent of this data element is to identify patients who have an active diagnosis of newborn live born. Using the Quality Data Model, this value set is used with \"Diagnosis\" category or the attributes \"Principal Diagnosis\" or \"Diagnoses\" attributes of the \"Encounter\" category.),(Inclusion Criteria: This value set includes ICD-10-CM  and SNOMED CT value sets.),(Exclusion Criteria: None.)"
```
Comment (okeefm) [inline cimpl/ecqm_dataelement.txt:5228]: Same as above with the double quotes. This aligns better with the other elements, which use single quotes, as well.

```suggestion
Description: "(Clinical Focus: This set of values contains diagnoses that are commonly used in the inpatient setting for newborn live born, and born in hospital. This value set is a grouping value set composed of  ICD-10-CM and SNOMED CT value sets.),(Data Element Scope: The intent of this data element is to identify patients who have an active diagnosis of newborn live born. Using the Quality Data Model, this value set is used with 'Diagnosis' category or the attributes 'Principal Diagnosis' or 'Diagnoses' attributes of the 'Encounter' category.),(Inclusion Criteria: This value set includes ICD-10-CM  and SNOMED CT value sets.),(Exclusion Criteria: None.)"
```

File: cimpl/ecqm_dataelement.txt [hunk 4]
```diff
@@ -6552,7 +6552,7 @@ Subject value is type Paroxetine
...
-Description: "(Clinical Focus: This set of values contains findings that indicate the normal findings on hearing examination or referral is needed. This value set contains SNOMED CT codes.),(Data Element Scope: The intent of this data element is to identify patients who have received newborn hearing screening with result of pass or refer, mapped to QDM attribute of result.),(Inclusion Criteria: Includes SNOMED CT codes. Includes normal finding on hearing examination and referral.),(Exclusion Criteria: None.)"
+Description: "(Clinical Focus: This set of values contains findings that indicate the normal findings on hearing examination or referral is needed. This value set contains SNOMED CT codes.),(Data Element Scope: The intent of this data element is to identify newly born patients who have received a complete hearing screening with result of pass or refer. Using the Quality Data Model, this value set is used with the “Result” attribute of the “Diagnostic Study” category.  ),(Inclusion Criteria: Include codes that represent normal findings on a hearing examination and indicate that a referral is needed. The value set includes SNOMED CT codes. ),(Exclusion Criteria: None.)"
```
Comment (okeefm) [inline cimpl/ecqm_dataelement.txt:6555]: Replacing smart quotes with single quotes.

```suggestion
Description: "(Clinical Focus: This set of values contains findings that indicate the normal findings on hearing examination or referral is needed. This value set contains SNOMED CT codes.),(Data Element Scope: The intent of this data element is to identify newly born patients who have received a complete hearing screening with result of pass or refer. Using the Quality Data Model, this value set is used with the 'Result' attribute of the 'Diagnostic Study' category.  ),(Inclusion Criteria: Include codes that represent normal findings on a hearing examination and indicate that a referral is needed. The value set includes SNOMED CT codes. ),(Exclusion Criteria: None.)"
```

File: cimpl/ecqm_dataelement.txt [hunk 5]
```diff
@@ -7316,7 +7316,7 @@ Subject value is type PsychVisitPsychotherapy
...
+Description: "(Clinical Focus: This grouping value set identifies diagnoses that are commonly used in the inpatient setting for psychiatric and mental health disorders.),(Data Element Scope: The intent of this data element is to identify patients who have an active diagnosis for a psychiatric or mental health disorder. Using the Quality Data Model, this value set is used with \"Principal Diagnosis\"  attribute of the \"Encounter\" category.),(Inclusion Criteria: Include codes representing a principal diagnosis of a psychiatric or mental health disorder. This value set includes ICD-10-CM and SNOMED CT codes.),(Exclusion Criteria: None.)"
```
Comment (okeefm) [inline cimpl/ecqm_dataelement.txt]: Replacing escaped double quotes with single quotes.
```suggestion
Description: "(Clinical Focus: This grouping value set identifies diagnoses that are commonly used in the inpatient setting for psychiatric and mental health disorders.),(Data Element Scope: The intent of this data element is to identify patients who have an active diagnosis for a psychiatric or mental health disorder. Using the Quality Data Model, this value set is used with 'Principal Diagnosis'  attribute of the 'Encounter' category.),(Inclusion Criteria: Include codes representing a principal diagnosis of a psychiatric or mental health disorder. This value set includes ICD-10-CM and SNOMED CT codes.),(Exclusion Criteria: None.)"
```

**Review by okeefm — CHANGES_REQUESTED** [review]
A few changes, mostly related to quotation marks.

**Review by okeefm — APPROVED** [review]

**Comment by mlterryMitre-zz** [comment]
@ssayer - Checked grammar looks good overall per request.  Value sets validation not checked to limited turnaround time.

**Pull request merged**